integrations/elasticsearch/src/elasticsearch_haystack/bm25_retriever.py

```diff
@@ -20,6 +48,24 @@ def __init__(
         top_k: int = 10,
         scale_score: bool = False,
     ):
+        """
+        Initialize ElasticsearchBM25Retriever with an instance ElasticsearchDocumentStore.
+
+        :param document_store: An instance of ElasticsearchDocumentStore.
+        :type document_store: ElasticsearchDocumentStore
+        :param filters: Filters applied to the retrieved Documents, for more info
+                        see `ElasticsearchDocumentStore.filter_documents`, defaults to None
+        :type filters: Optional[Dict[str, Any]], optional
+        :param fuzziness: Fuzziness parameter passed to Elasticsearch, defaults to "AUTO".
+                          see the official documentation for valid values:
+                          https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#fuzziness
```

Comment:
Shouldn't we be following the [RST syntax](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages)?

Reply:
Good point. ☝️

Reply:
Now that I think about it this should be valid RST syntax, the link will be highlighted and threated as such. 🤔

Reply:
I'll take your word for it 👍  

That being said, we might want to discuss elsewhere if using hyperlinks (as opposed to bare URLs) improves readability. And on a related note, we should also add a CI workflow that automatically deploys documentation changes to a preview service